### PR TITLE
[TRANSLATION] Basque and Spanish new files and fixes

### PR DESCRIPTION
**Pull request by julenuri**
_BASQUE_
* Usetup translation in Euskara (Basque) Lang.
* Mspaint, Notepad and Acppage fixed.

_SPANISH_
* Fixes in dll/Win32 Spanish .rc that where wrong.
* Tapiui Spanish created.
* Msi translated

Ready to merge when you want.

### CODE REVIEW

**Comment by HBelusca** [comment]
I confirm that, according to MSDN, Basque language is identified by:
(0x042D = 1069) eu-ES

File: base/setup/usetup/muilanguages.h [hunk 1]
```diff
@@ -200,8 +200,8 @@
 #ifdef LANGUAGE_ET_EE
     #include "lang/et-EE.h"
 #endif
-// #ifdef LANGUAGE_EU_ES
-    // #include "lang/eu-ES.h"
+#ifdef LANGUAGE_EU_ES
+    #include "lang/eu-ES.h"
 // #endif
```
Comment (learn-more) [inline base/setup/usetup/muilanguages.h:205]: You forgot to uncomment this line.
Please test your translation changes!

Comment (julenuri) [reply base/setup/usetup/muilanguages.h:205]: Sorry, I uploaded a version of the file in which one I haven't uncommented the line. It's a draft yet.

**Comment by julenuri** [comment]
It's mostly ready.

**Comment by julenuri** [comment]
I won't do any more changes here.

**Comment by HBelusca** [comment]
@julenuri You know that deleting your branch doesn't help us for fixing what needs to be fixed in your PR...

**Comment by julenuri** [comment]
@HBelusca  No, I didn't. Sorry, I'm not familiar with "how to do things" in Github. But, I think I can close this and restore the old branch files and make another?

**Comment by HBelusca** [comment]
OK I've found a possibility to do the fix by myself, based on https://webapps.stackexchange.com/questions/124056/can-you-add-to-a-github-pull-request-after-the-source-repo-is-deleted

**Comment by HBelusca** [comment]
The PR has been manually fixed and merged. Thanks for your contribution!

**Comment by julenuri** [comment]
You welcome and sorry for bothering.

**Pull request closed without merging**